pkg/printers/internalversion/printers.go

```diff
@@ -60,7 +59,20 @@ import (
 	"k8s.io/kubernetes/pkg/util/node"
 )
 
-const loadBalancerWidth = 16
+const (
+	loadBalancerWidth = 16
+
+	// LabelNodeRoleMaster specifies that a node is a master
+	// It's copied over to here until it's merged in core: https://github.com/kubernetes/kubernetes/pull/39112
+	LabelNodeRoleMaster = "node-role.kubernetes.io/master"
```

Comment:
Is there a doc somewhere else that clarifies why a boolean "is master" label (a valueless key) is better than the enum-ish "role" label below?